src/libfetchers/git.cc

```diff
@@ -47,6 +47,8 @@ struct GitInputScheme : InputScheme
                 attrs.emplace(name, value);
             else if (name == "shallow")
                 attrs.emplace(name, Explicit<bool> { value == "1" });
+            else if (name == "largeFileSystem")
```

Comment:
Comparing a "1" string seems pretty bad. Should creating a function that parse a string as boolean (like parse true or false as valid entry) be a good idea ? A user might think putting ?largeFileSystem=true work, then be surprised (but the shallow code did that).